python/mooseutils/csvdiff.py

```diff
@@ -289,47 +290,53 @@ def diff(self):
             for val1, val2 in zip( table1[key], table2[key] ):
                 # if customized tolerances specified use them otherwise
                 # use the default
-                if self.custom_columns:
-                    try:
-                        abs_zero = abs_zero_map[key]
-                    except:
-                        abs_zero = self.abs_zero
-                if abs(val1) < abs_zero:
-                    val1 = 0
-                if abs(val2) < abs_zero:
-                    val2 = 0
-
-                # disallow nan in the gold file
-                if math.isnan(val1):
-                    self.addError(self.files[0], "The values in column \"" + key.strip() + "\" contain NaN")
-
-                # disallow inf in the gold file
-                if math.isinf(val1):
-                    self.addError(self.files[0], "The values in column \"" + key.strip() + "\" contain Inf")
-
-                # if they're both exactly zero (due to the threshold above) then they're equal so pass this test
-                if val1 == 0 and val2 == 0:
-                    continue
-
-                rel_diff = 0
-                if max( abs(val1), abs(val2) ) > 0:
-                    rel_diff = abs( ( val1 - val2 ) / max( abs(val1), abs(val2) ) )
-
-                # if customized tolerances specified use them otherwise
-                # use the default
-                if self.custom_columns:
-                    try:
-                        rel_tol = rel_err_map[key]
-                    except:
-                        rel_tol = self.rel_tol
-                if rel_diff > rel_tol:
-                    self.addError(self.files[1], "The values in column \"%s\" don't match. \n\trelative diff:   %.3e ~ %.3e = %.3e (%.3e)" % (key.strip(),
+                if self.ignore is not None:
```

Comment:
can you avoid the large diff here?

You could do a
if self.ignore:
  continue